tests/test_command_arf_to_graph.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import os
+import pexpect
```

Comment:
If `pexpect` is not available, tests that require it should be skipped, as you have majority of tests that don't require `pexpect`. Check out `pytest.skipif`: https://docs.pytest.org/en/latest/skipping.html#id1